### DISCUSSION TITLE
How to update Opportunities Actual date & Actual sale?

### DISCUSSION THREAD

**Discussion by tquoc229** [YetiForceCompany/YetiForceCRM#16885] [How to]
Hi, 
I want to update Opportunities Actual date & Actual sale but these fields were disabled. How can I update these fields?
![image](https://user-images.githubusercontent.com/78257/218438407-94e37f5d-b51a-44b0-9a75-773c19e7dae2.png)

**Comment by tquoc229** [comment]
Is there anyone know about this?

**Reply by apcloic** [reply]
@tquoc229 
Maybe this field is not configured by default and require some customization ?
#4402

**Comment by vovpff** [comment]
I think this fields filled automatically when some conditions are met. You can made this fields writable trough admin panel

**Comment by deepti4666** [comment]
You can do this via the Automation > Workflows
